easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.3.1-fosscuda-2020a-Python-3.8.2.eb

```diff
@@ -119,8 +119,8 @@ exts_list = [
         'unpack_sources': False,
         'checksums': ['b75e034300ccb169403cf2695adf3368da68863aeb0c14c3760064c713d5c486'],
     }),
-    ('astor', '0.8.0', {
-        'checksums': ['37a6eed8b371f1228db08234ed7f6cfdc7817a3ed3824797e20cbb11dc2a7862'],
+    ('astor', '0.8.1', {
```

Comment:
@Flamefire Is there a specific need for the version bump? If not, should we avoid the in-place update?

Reply:
Yes: it needs the bugfix for the new setuptools we now use. See @Micket report in Slack and https://github.com/berkerpeksag/astor/commit/2159cfeb905af82eaf0cdf6beaa5e7b486a71c86

TLDR: Us correctly installing setuptools now requires the new astor

Reply:
I think this is fine to change: Those who have built the old version from before can just continue to use that, anyone building a new one now would fail, so we should fix this.

Reply:
Yeah, see my report above: https://gist.github.com/Flamefire/b9c61028995b9e5f75e992325bb470db

Reply:
OK, understood, thanks for clarifying!